exercises/takudzwa_makoni/ex01_basics/src/exercise01.cpp

```diff
@@ -0,0 +1,100 @@
+
+#include "pch.h"
+#include <iostream>
+#include <fstream>
+#include <list>
+#include <iomanip>
+#include <string>
+#include <algorithm>
+#include <cctype>
+#include <iomanip>
+
+
+using namespace std;
```

Comment:
In Mantid we avoid `using namespace`s that are not our own.

Reply:
have made these changes